docs/IntegrationWithExistingApps.md

```diff
@@ -720,6 +720,8 @@ public boolean onKeyUp(int keyCode, KeyEvent event) {
 
 That's it, your activity is ready to run some JavaScript code.
 
+> If your app is targeting the Android `api level 23` or greater, make sure you have, for the development build, the `overlay permission` enable. You can check it with `Settings.canDrawOverlays(this);`. 
```

Comment:
What I mean is you should explain why you have to enable this. You mention it in your commit summary.